File: 프로그래머스/lv0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.java
class Solution {
  public int[] solution(int[] array) {

        int maxValue = array[0];
        int countIndex = 0;

        for (int i : array) {
            if (i > maxValue) {
                maxValue=i;
            }
        }
      
        for (int i : array) {
            if(i==maxValue){
                break;
            }
            countIndex++;
        }
        
        int[] answer ={maxValue,countIndex};

        return answer;
    }
}

File: 프로그래머스/lv0/120899. 가장 큰 수 찾기/가장 큰 수 찾기.py
def solution(array):
    max_value = array[0]
    count_index =0
    for a in array:
        if max_value < a:
            max_value =a
    
    for a in array:
        if max_value !=a:
            count_index +=1
        else:
            break
            
    result=[max_value,count_index]
    
    return result